inc/saiacl.h

```diff
@@ -3221,10 +3224,21 @@ typedef enum _sai_acl_entry_attr_t
      */
     SAI_ACL_ENTRY_ATTR_ACTION_CHAIN_REDIRECT = SAI_ACL_ENTRY_ATTR_ACTION_START + 0x38,
 
+    /**
+     * @brief Disable packet trimming for a given match condition.
+     *
+     * This rule takes effect only when packet trimming is configured on a TC to which a packet belongs.
```

Comment:
Please change "configured on a TC" to "configured on a buffer profile".

Reply:
Done